docs/basics/101-130-yodaproject.rst

```diff
@@ -0,0 +1,763 @@
+
+   .. figure:: ../artwork/src/screenshot_midtermproject.png
+      :alt: The midterm project repository, published to Github
+
+   The above screenshot shows the linkage between the analysis project you will create
+   and its subdataset. Clicking on the subdataset (highlighted) will take you to the iris dataset
+   the handbook provides, shown below.
+
+   .. figure:: ../artwork/src/screenshot_submodule.png
+      :alt: The input dataset is linked
+
+.. note::
+
+   The upcoming part requires a GitHub account. If you do not have one you
+   can either
+
+   - Create on now -- it is fast, free, and you can get rid of it afterwards,
+     if you want to.
+   - Or exchange the command ``create-sibling-github`` with
+     ``create-sibling-gitlab`` if you have a GitLab account instead of a GitHub
+     account.
+   - Don't listen to me and not follow along. I'm only a book, not your mom.
+
+For this, you need to
+
+- create a repository for this dataset on GitHub,
+- configure this GitHub repository to be a :term:`sibling` of the ``midterm_project`` dataset,
+- and *publish* your dataset to GitHub.
+
+Luckily, DataLad can make all of this very easy with the
+:command:`datalad create-sibling-github` (:manpage:`datalad-create-sibling-github` manual)
+command (or, for `GitLab <https://about.gitlab.com/>`_, :command:`datalad create-sibling-gitlab`,
+:manpage:`datalad-create-sibling-gitlab` manual).
+
+.. index:: ! datalad command; create-sibling-github
+.. index:: ! datalad command; create-sibling-gitlab
+
+The command takes a repository name and GitHub authentication credentials
+(either in the command line call with options ``github-login <NAME>`` and
+``github-passwd <PASSWORD>``, with an *oauth* token stored in the Git
+configuration [#f5]_, or interactively). Based on the credentials and the
+repository name, it will create a new, empty repository on GitHub, and
+configure this repository as a sibling of the dataset. If the ``-r/--recursive``
+option is specified, siblings will be created *recursively*, i.e., not only a
+repository for the ``midterm_project`` dataset will be created, but also one
+for the ``iris_data`` dataset:
+
+
+.. ifconfig:: internal
+
+    .. runrecord:: _examples/DL-101-130-116
+       :language: console
+       :workdir: dl-101/DataLad-101/midterm_project
+       :realcommand: datalad --log-level critical siblings add -d . --name github --url https://github.com/adswa/midtermproject.git
+
+.. code-block:: bash
+
+   $ datalad create-sibling-github -d . midtermproject
+   .: github(-) [https://github.com/adswa/midtermproject.git (git)]
+   .: github(-) [https://github.com/adswa/midtermproject-input.git (git)]
+   'https://github.com/adswa/midtermproject.git' configured as sibling 'github' for <Dataset path=/home/me/dl-101/DataLad-101/midterm_project>
+   'https://github.com/adswa/midtermproject-input.git' configured as sibling 'github' for <Dataset path=/home/me/dl-101/DataLad-101/midterm_project/input>
```

Comment:
This is still the old output from recursive mode used before. Should be sufficient to kill the '-input' lines.

Reply:
thx for catching that!